app/build.gradle

```diff
@@ -84,12 +84,19 @@ dependencies {
     //test implementations
     testImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.0'
     testImplementation "io.mockk:mockk:1.12.2"
-    testImplementation 'junit:junit:4.+'
+    testImplementation 'junit:junit:'
     testImplementation 'androix.arch.core:core-testing:2.1.0'
 
     //Koin for Android
     implementation "io.insert-koin:koin-android:3.1.4"
     implementation 'com.squareup.okhttp3:okhttp:4.9.2'
-    implementation "androidx.core:core-ktx:+"
+    implementation "androidx.core:core-ktx:1.7.0"
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+
+    //fix "Gradle Sync Issues"
+    implementation 'com.squareup.retrofit2:retrofit:2.8.2'
+    implementation 'com.google.code.gson:gson:2.8.6'
+    implementation 'com.squareup.retrofit2:converter-gson:2.8.2'
+    implementation 'com.squareup.okhttp3:logging-interceptor:3.4.1'
+    implementation 'com.squareup.okhttp3:okhttp:4.9.2'
```

Comment:
In order to successfully debug, I checked the contents of the error report to solve the "caused by: org.gradle.internal.resolve.moduleversionnotfoundexception: could not find com.squareup.retrofit2: retrofit2: 2.7.1
”Wait for a question, I'm based on“ https://stackoverflow.com/questions/52215153/unable-to-resolve-dependency-for-retrofit2 ”The version has been revised according to your comments